src/map.cpp

```diff
@@ -4936,6 +5022,11 @@ item &map::add_item( const tripoint &p, item new_item )
     if( !inbounds( p ) ) {
         return null_item_reference();
     }
+
+    if( !check_for_spawn_chance( new_item ) ) {
+        return null_item_reference();
+    }
```

Comment:
This is most definitely the wrong way to check for spawn chances, since it's (indirectly) called by pretty much everything, like dropping things on the floor etc.

Reply:
Yeah, you're right. I didn't test this case.

Reply:
[6268d2c](https://github.com/CleverRaven/Cataclysm-DDA/pull/62583/commits/6268d2c5b88848d98fb0c81de68cf7e88d32811d) should fix this.